DIVA-DIA/DIVA-DAF
# Save metrics to csv

### Description
Save results into a machine readable format.

### How to Test/Run?
`python run.py`
in the output folder should exist a `csv` folder.
or in any other branch:
`python run.py logger=csv.yaml`

### Something missing?
nope
